Source/SwiftLintFramework/Models/Configuration.swift

```diff
@@ -47,6 +48,8 @@ public struct Configuration: Equatable {
     public let rules: [Rule]
     public var rootPath: String?              // the root path to search for nested configurations
     public var configurationPath: String?     // if successfully loaded from a path
+    public var hash: Int?
```

Comment:
Yeah, ok thinking more about it, `Hashable` implies `Equatable`, and the protocol semantics require that all user-visible data from two equal types be indistinguishable, so makes sense not to conform to it.